dashboard/test/ui/support/hooks.rb

```diff
@@ -11,3 +11,11 @@
     saw_failure = true
   end
 end
+
+Before('@as_student') do
+  steps 'Given I manually sign in as "TestStudent"'
```

Comment:
Yup it is - you can ignore my comment then

On Wed, Nov 18, 2015 at 3:32 PM, Brad Buchanan <notifications@github.com>
wrote:

> In dashboard/test/ui/support/hooks.rb
> <https://github.com/code-dot-org/code-dot-org/pull/5493#discussion_r45278015>
> :
>
> > @@ -11,3 +11,11 @@
> >      saw_failure = true
> >    end
> >  end
> > +
> > +Before('@as_student') do
> > +  steps 'Given I manually sign in as "TestStudent"'
>
> Mehal, I think that sleep is already included in the I manually sign in as
> step definition
> <https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/step_definitions/steps.rb#L491>.
> Can you confirm that the 2-second wait there is what you're looking for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/5493/files#r45278015>.
>



-- 
Mehal Shah